README.md

```diff
@@ -31,11 +31,11 @@ trait. Here is a basic example using `File` as a backend:
 /// A simple wrapper for a device node.
 pub struct Card(std::fs::File);
 
-/// Implementing `AsRawFd` is a prerequisite to implementing the traits found
-/// in this crate. Here, we are just calling `as_raw_fd()` on the inner File.
+/// Implementing `AsFd` is a prerequisite to implementing the traits found
+/// in this crate. Here, we are just calling `as_fd()` on the inner File.
```

Comment:
Making these and the documentation below intradoc links might be nice?

Reply:
This is a comment in a code example in the `README`, so it can't really be a working link to anything. Similarly with the comment in `examples/` (is there any option that will make `cargo doc` document examples?).

Reply:
> is there any option that will make `cargo doc` document examples?

Pretty sure `rust-analyzer` can show these, and maybe with `--all-targets` + `--document-private-items`?